lang/en.json

```diff
@@ -1,558 +1,549 @@
-      "GettingStartedVariableComponent": "Variables - Lets you retrieve data which can be used by tags and triggers.",
-      "GettingStartedWhyDoINeed": "Why or when do I need a Tag Manager?",
-      "GettingStartedWhyMakesLifeEasier": "A Tag Manager makes your life easier when you want to modify any of these snippets on your website as you will no longer need a developer to make the needed changes for you. Instead of waiting for someone to make these changes and to deploy your website, you can now easily make the needed changes yourself. This lets you not only bring changes to the market faster, but also reduces cost.",
-      "GettingStartedWhyThirdPartySnippets": "It also comes in handy if you embed many third-party snippets into your website and want to bring in some order to oversee all the snippets that are embedded and have a convenient way to manage them.",
-      "GettingStartedWhyAccuracyPerformance": "The Tag Manager also makes sure all that all snippets are implemented correctly and loaded in the right way for faster performance.",
-      "GettingStartedHowDoI": "How do I get started?",
-      "GettingStartedHowCreateContainer": "If you haven't created a container yet, %1$screate a container now%2$s.",
-      "GettingStartedHowCopyCode": "Next you need to copy/paste the code for the container into your website, this is a simple HTML snippet. From this point, this code will load all other snippets and you usually won't need to make any changes to your website anymore.",
-      "GettingStartedHowAddTagsToContainer": "Now you can add one or multiple tags to your container. If you have embedded a tag manually into your site in the past, you should at the same time also remove all previously added snippets from your website as they will be then loaded through the tag manager.",
-      "GettingStartedWhatIfUnsupported": "What if a tag, trigger, or variable I need is not supported yet?",
-      "GettingStartedCustomTags": "There are custom tags, triggers, and variables available to let you implement pretty much any use case you need.",
-      "GettingStartedContributeTags": "If you use features regularly which are not available yet, or you have a product you want to integrate into the Tag Manager, please check out our %1$sdeveloper documentation%2$s on how to add your own tags, triggers, and variables. It is really easy.",
-      "GoogleAnalyticsUniversalTagDescription": "Google Analytics is a web analytics service offered by Google.",
-      "GoogleAnalyticsUniversalTagHelp": "This tag lets you track website pageviews into your Google Analytics. To obtain the Property-ID please log in to your Google Analytics account.",
-      "GoogleAnalyticsUniversalTagName": "Google Analytics (Universal)",
-      "HideAdvancedSettings": "Hide advanced settings",
-      "HistoryChangeTriggerDescription": "Triggered when the current URL changes.",
-      "HistoryChangeTriggerName": "History Change",
-      "HistoryHashNewPathVariableDescription": "",
```

Comment:
These lines should not be removed (I guess the script did it because they are empty

Reply:
@Findus23 should be fixed.